src/me/StevenLawson/TotalFreedomMod/Commands/Command_savinghelp

```diff
@@ -0,0 +1,34 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.ONLY_IN_GAME)
+@CommandParameters(description = "Clears a player's chat, leave blank to clear public chat.", usage = "/<command> <playername>")
+public class Command_cc extends TFM_Command
+{
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        
+        if (args.length == 0)
```

Comment:
@Wild1145 It doesn't matter, but this is the shortest (and cleanest) way of achieving what's required.